docs/machine-learning/tutorials/object-detection-model-builder.md

```diff
@@ -0,0 +1,263 @@
+
+![Model Builder Evaluate Step](./media/object-detection-model-builder/obj-det-evaluate.png)
+
+In this case, the accuracy says 100%, which means that the model is more than likely *overfit* due to too few images in the dataset.
+
+You can use the **Try your model** experience to quickly check whether your model is performing as expected.
+
+Select **Browse an image** and provide a test image, preferably one that the model did not use as part of training.
+
+![Model Builder Try your model](./media/object-detection-model-builder/obj-det-try-model.png)
+
+The score shown on each detected bounding box indicates the **confidence** of the detected object. For instance, in the screenshot above, the score on the bounding box around the stop sign indicates that the model is 99% sure that the detected object is a stop sign.
+
+The **Score threshold**, which can be increased or decreased with the threshold slider, will add and remove detected objects based on their scores. For instance, if the threshold is .51, then the model will only show objects that have a confidence score of .51 or above. As you increase the threshold, you will see less detected objects, and as you decrease the threshold, you will see more detected objects.
+
+If you're not satisfied with your accuracy metrics, one easy way to try to improve model accuracy is to use more data. Otherwise, select the **Next step** link to move on to the **Code** step in Model Builder.
+
+## Add the code to make predictions
+
+Two projects are created as a result of the training process.
+
+- *StopSignDetectionML.ConsoleApp*: A C# .NET Core Console application that contains the model training code and sample consumption code.
+- *StopSignDetectionML.Model*: A .NET Standard class library containing the data models that define the schema of input and output model data, the saved version of the best performing model during training, and a helper class called `ConsumeModel` to make predictions.
+
+1. In the code step of Model Builder, select **Add Projects** to add the auto-generated projects to the solution.
+1. Open the *Program.cs* file in the *StopSignDetection* project, and add the following using statement at the top of the file to reference the *StopSignDetectionML.Model* project:
+
+    [!code-csharp [ProgramUsings](~/machinelearning-samples/samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/Program.cs#L2)]
+
+1. Download the following [test image](~/machinelearning-samples/samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/test-image1.jpeg).
+1. Create a new instance of the `ModelInput` class with the `ImageSource` property set to the filepath of your test image inside the `Main` method of your application. Replace the "Hello World" statement with the following code:
+
+    [!code-csharp [InputData](~/machinelearning-samples/samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/Program.cs#L11-L15)]
```

Comment:
The ModelInput code I think is cut off here. Just make sure the right lines are referenced.